### DISCUSSION TITLE
How do I delete a package？？

### DISCUSSION THREAD

**Discussion by BubbleM** [nestdotland/nest.land#306] [Q&A] [answered]
First publish，accidentally posted as 2.0.0
Can I delete or undo it ？

**Comment by maximousblk** [comment]
nest.land by design restricts anyone from deleting the modules or any version of it. So no you cannot delete or undo the published version.

**Reply by tomByrer** [reply]
That's going to be a problem for malicious code, exposing secrets, etc....
Malicious code has happened only a few times in npm's history, but delete was convenient there.
Exposing secrets; much more common than you think.

**Comment by t8** [comment, marked as answer]
Once data is uploaded to the blockweave, it’s instantly replicated across
100+ nodes incentivized to store this data forever.

We can unlist (hide) modules, but we cannot remove them. Power lies with
the developer, as it should.

**Reply by BubbleM** [reply]
Is there any way I can hide this module ：https://nest.land/package/umajs_deno ？

I republished a new module，This old module is no longer maintained

**Reply by SteelAlloy** [reply]
If you want a module to be no longer visible in the gallery, just add the `unlisted` field to your configuration file.
```json
// egg.json
{
  "unlisted": true
}
```
You then need to publish a new version of the module to update the configuration. 👍

In the future it will be easier, you will just have to change it on the website or synchronize the configuration. 🎇

**Reply by BubbleM** [reply]
it's great，tks
